docker/build_srt_xtransmit.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+git submodule update --init --recursive
+if [[ ! -z "${SRT_VERSION}" ]];then 
+  cd submodule/srt
+  git checkout "${SRT_VERSION}"
+  cd ../..
+fi
+mkdir -p _build
+cd _build
+cmake ../ ${BUILD_OPTIONS}
```

Comment:
Deepsource complaint: Use double quote to prevent globbing and word splitting.
```suggestion
cmake ../ "${BUILD_OPTIONS}"
```